homeassistant/components/androidtv/media_player.py

```diff
@@ -319,7 +323,7 @@ def __init__(self, aftv, name, apps, turn_on_command, turn_off_command):
 
         # Property attributes
         self._adb_response = None
-        self._available = self.aftv.available
+        self._available = True
```

Comment:
This check was unnecessary because we already check it in the `setup_platform` function, and if it's not available then we don't create the `MediaPlayerDevice` instance: https://github.com/home-assistant/home-assistant/blob/d7c15f7189b107b0c0429fc7570d0a01a555d453/homeassistant/components/androidtv/media_player.py#L180